src/Gui/ViewProviderGeometryObject.cpp

```diff
@@ -288,14 +271,33 @@ unsigned long ViewProviderGeometryObject::getBoundColor() const
     return bbcol;
 }
 
-void ViewProviderGeometryObject::setSoMaterial(const App::Material& source)
+void ViewProviderGeometryObject::setCoinAppearance(const App::Material& source)
 {
+#if 0
+    if (!source.image.empty()) {
+        Base::Console().Log("setCoinAppearance(Texture)\n");
+        activateTexture2D();
+
+        QByteArray by = QByteArray::fromBase64(QString::fromStdString(source.image).toUtf8());
+        auto image = QImage::fromData(by, "PNG");  //.scaled(64, 64, Qt::KeepAspectRatio);
+
+        SoSFImage texture;
+        Gui::BitmapFactory().convert(image, texture);
+        pcShapeTexture2D->image = texture;
+    } else {
+        Base::Console().Log("setCoinAppearance(Material)\n");
+        activateMaterial();
+    }
+#endif

```

Comment:
Keeping this for a reason?

Reply:
Yes. It will be used by the texture code